src/ZMQ.jl

```diff
@@ -501,8 +501,8 @@ function send(socket::Socket, zmsg::Message, flag=int32(0))
     if (get_events(socket) & POLLOUT) == 0
         wait(socket; writable = true)
     end
-    rc = ccall((:zmq_msg_send, zmq), Cint, (Ptr{Void}, Ptr{Message}, Cint),
-                &zmsg, socket.data, flag)
+    rc = ccall((:zmq_msg_send, zmq), Cint, (Ref{Message}, Ptr{Message}, Cint),
+                zmsg, socket.data, flag)
```

Comment:
this wasn't supposed to be deprecated (yet). although i see the argument types were reversed (http://api.zeromq.org/3-3:zmq-msg-send), passing `&arg` to `Ptr{Void}` was supposed to keep working